server/api/api.go

```diff
@@ -1120,6 +1121,52 @@ func (a *API) handleUploadFile(w http.ResponseWriter, r *http.Request) {
 	jsonBytesResponse(w, http.StatusOK, data)
 }
 
+func (a *API) getWorkspaceUsers(w http.ResponseWriter, r *http.Request) {
+	// swagger:operation GET /api/v1/workspaces/{workspaceID}/users getWorkspaceUsers
+	//
+	// Returns workspace users
+	//
+	// ---
+	// produces:
+	// - application/json
+	// parameters:
+	// - name: workspaceID
+	//   in: path
+	//   description: Workspace ID
+	//   required: true
+	//   type: string
+	// security:
+	// - BearerAuth: []
+	// responses:
+	//   '200':
+	//     description: success
+	//     schema:
+	//       type: array
+	//       items:
+	//         "$ref": "#/definitions/User"
+	//   default:
+	//     description: internal error
+	//     schema:
+	//       "$ref": "#/definitions/ErrorResponse"
+
+	vars := mux.Vars(r)
+	workspaceID := vars["workspaceID"]
+
+	users, err := a.app().GetWorkspaceUsers(workspaceID)
```

Comment:
Probably we should check if you have permissions to see the workspace.

Reply:
Done.